sources/tools/Stride.VisualStudio.Package/Commands/StrideCommandsProxy.cs

```diff
@@ -188,7 +188,7 @@ internal static async Task<PackageInfo> FindStrideSdkDir(string solution, string
                 }
                 if (!success)
                 {
-                    throw new InvalidOperationException($"Could not restore {packageName} {packageInfo.ExpectedVersion}, this visual studio extension may fail to work properly without it. To fix this you can either build {packageName} or pull the right version from nugget manually");
+                    throw new InvalidOperationException($"Could not restore {packageName} {packageInfo.ExpectedVersion}, this visual studio extension may fail to work properly without it. To fix this you can either build {packageName} or pull the right version from nuget manually path {Path.GetDirectoryName(solution)}");
```

Comment:
I think it should be "NuGet". Minor nitpick 😁

Reply:
fixed also removed my debug path test